### PR TITLE
Create XXXX-optimizing-query-fees-sent-to-indexers.md

### PR DESCRIPTION
**Pull request by howard-the-graph**

### CODE REVIEW

File: gips/XXXX-optimizing-query-fees-sent-to-indexers.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+GIP: "XXXX"
```
Comment (pcarranzav) [inline gips/XXXX-optimizing-query-fees-sent-to-indexers.md:2]: @howard-the-graph let's self-assign a GIP number here, 0063 is the next available

Comment (pcarranzav) [reply gips/XXXX-optimizing-query-fees-sent-to-indexers.md:2]: (This is the proposed new process, which is pending official approval but some of which we've already been doing for a while: https://github.com/graphprotocol/graph-improvement-proposals/pull/30 )

File: gips/XXXX-optimizing-query-fees-sent-to-indexers.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Title: Optimizing Query Fees sent to Indexers
+Authors: Howard
```
Comment (pcarranzav) [inline gips/XXXX-optimizing-query-fees-sent-to-indexers.md:4]: The template requires adding contact information here, could you add an email in angle brackets?

File: gips/XXXX-optimizing-query-fees-sent-to-indexers.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Authors: Howard 
+Created: 2023-11-08
+Updated: 2023-11-08
+Stage: Draft
+Category: Economic Parameters
+---
+
+## Abstract
+
+Today, at least 11% of query fees paid by data consumers do not ever make it to either the Gateway or the Indexers that provide the service to the data consumers. This GIP aims to reduce these transactional costs. This can be done without changing any smart contract code; instead, only two parameters need to be tweaked: removing the fees sent to curators and setting the protocol tax to zero. This will simplify how the protocol operates and potentially save GRT both for developers and Indexers.
```
Comment (MichaelMacaulay) [inline gips/XXXX-optimizing-query-fees-sent-to-indexers.md:13]: ~~Minor nitpick, but it is preferred to capitalize all network roles (Curators does not seem to be capitalized).~~  

Edit: after reading through, it strikes me as a good stylistic choice _not_ to capitalize curators. Great work!

**Review by MichaelMacaulay — APPROVED** [review]
Amazing work! Very well-written!

File: gips/XXXX-optimizing-query-fees-sent-to-indexers.md (hunk 1 above)
Comment (howard-the-graph) [reply gips/XXXX-optimizing-query-fees-sent-to-indexers.md]: Ah, good to know. Updated ✅

File: gips/XXXX-optimizing-query-fees-sent-to-indexers.md (hunk 2 above)
Comment (howard-the-graph) [reply gips/XXXX-optimizing-query-fees-sent-to-indexers.md]: Done ✅

**Pull request merged**